data/json/recipes/other/tool.json

```diff
@@ -177,5 +177,26 @@
     "book_learn": [ [ "mag_survival", 2 ], [ "textbook_survival", 1 ], [ "cookbook", 2 ] ],
     "qualities": [ { "id": "CUT", "level": 1 } ],
     "components": [ [ [ "can_food_unsealed", 2 ], [ "scrap", 4 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "razor_blade",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "skills_required": [ "fabrication", 3 ],
+    "difficulty": 3,
+    "time": 8000,
+    "autolearn": true,
+    "book_learn": [
+      [ "mag_survival", 1 ],
+      [ "textbook_survival", 1 ],
+      [ "mag_fabrication", 1 ],
+      [ "manual_fabrication", 1 ],
+      [ "textbook_fabrication", 1 ]
+    ],
+    "qualities": [ { "id": "HAMMER", "level": 1 }, { "id": "SAW_M", "level": 1 } ],
+    "tools": [ [ [ "fire", -1 ] ] ],
```

Comment:
Would need forging tools; steel can't be melted by fire.

Reply:
No forging is being done. The steel is being mostly cut to shape.